scripts/contributing.js

```diff
@@ -0,0 +1,36 @@
+/* global jQuery*/
+
+(function($){
+    var githubIssuesEndpoint = 'https://api.github.com/repos/babel/babel/issues';
```

Comment:
May not have to be in this pr but we could actually link up issues in other repos too

babel/x such as babili, babel-preset-env, babylon, etc